Sources/ParseSwift/Coding/Extensions.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  File.swift
+//  
```

Comment:
Switch file name in comments to match file

Reply:
Good catch! I'll fix all of these up.